layouts/shortcodes/chapterstyle.html

```
@@ -1,4 +1,16 @@
-{{ $style := .Get "style" | default "" }}
-<div class="chapter-content"{{ with $style }} style="{{ . }}"{{ end }}>
-  {{ .Inner | .Page.RenderString }}
-</div>
+{{/*
+  Shortcode to wrap markdown content in a styled chapter container.
+  Usage:
+  {{< chapter >}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)